AuxTel/AuxTel-Troubleshooting/General-Troubleshooting/Images-out-of-focus.rst

```diff
@@ -0,0 +1,128 @@
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+Image out of focus
+#########################
+
+.. note::
+    This is a procedure template file that is associated with a template directory structure. This note should be deleted when the section is properly populated.
+
+.. _Image-out-of-focus-Procedure-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this Image out of focus procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This procedure should be used when images appear out of focus due to a WEP failure or other related issues. Users will notice images ranging from slightly out of focus to large donut appearances when hexapod offsets are not correctly applied.
+
+.. _Image-out-of-focus-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+- The focus script `latiss_wep_align` might fail, leaving the system out of focus.
+- Out-of-focus images can appear as large donuts when hexapod offsets are left in intra or extra hexapod positions due to script failure.
+
+.. figure:: ./_static/out-focus.png
+    :name: out-focus
+    :width: 400
+
+
+.. _Image-out-of-focus-Procedure-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. todo::
+   Make sure everything is in a safe or idle state before Image out of focus. Describe relevant safety steps if necessary.
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+This paragraph describes some general guidance for this procedure. This paragraph is optional depending on its usefulness and the complexity of the procedure.
+
+In the case of a very straightforward procedure, a simple numbered list could be used, like as follows:
```

Comment:
Remove this general comment too, or comment it out.

Reply:
done